File: punto.java
package punto;


public class Punto {
    private double x;
    private double y;

    public Punto(double x, double y) {
        this.x = x;
        this.y = y;
    }

    public void coordenadasCartesianas() {
        System.out.println("los puntos en el plano cartesiano son : (" + x + ", " + y + ")");
    }

    public void coordenadasPolares() {
        double c = Math.sqrt(x * x + y * y);
        double a = Math.atan2(y, x);
        System.out.println("Las coordenadas Polares son : (r = " + c + ", θ = " + a + ")");
    }

    public String toString() {
        return "(" + x + ", " + y + ")";
    }

    public static void main(String[] args) {
        Punto p = new Punto(3, 4);
        p.coordenadasCartesianas();
        p.coordenadasPolares();
        p.toString();
    }
}

File: punto.py
import math

class Punto:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def coordenadas_cartesianas(self):
        print(f"Los puntos del plano cartesiano son: ({self.x}, {self.y})")

    def coordenadas_polares(self):
        c = (self.x**2 + self.y**2)**(1/2)
        a = math.atan2(self.y, self.x)
        print(f"Las coordenadas polares son: (r = {c}, θ = {a})")

    def __str__(self):
        return f"({self.x}, {self.y})"


if __name__ == "__main__":
    p = Punto(3, 4)
    p.coordenadas_cartesianas()
    p.coordenadas_polares()
    p.__str__()
